src/registrar/tests/test_models.py

```diff
@@ -1495,11 +1495,26 @@ def test_create_or_update_organization_type_existing_instance_updates_election_b
         self.assertEqual(domain_request.is_election_board, False)
         self.assertEqual(domain_request.organization_type, DomainRequest.OrgChoicesElectionOffice.CITY)
 
-        # Try reverting setting an invalid value for election board (should revert to False)
+    @less_console_noise_decorator
+    def test_create_or_update_organization_type_existing_instance_updates_election_board_to_none(self):
+        """Test create_or_update_organization_type for an existing instance."""
+        domain_request = completed_domain_request(
+            status=DomainRequest.DomainRequestStatus.STARTED,
+            name="started.gov",
+            generic_org_type=DomainRequest.OrganizationChoices.CITY,
+            is_election_board=False,
+        )
+        domain_request.is_election_board = True
+        domain_request.save()
```

Comment:
That sounds like it would be testing the completed_domain_request function itself rather than the model. What would be the point of assigning is_election_board to false if the function would spit back a domain request where it updated to true before we even did anything?